Marlin/src/pins/stm32/pins_ARMED.h

```diff
@@ -117,6 +117,7 @@
 #define TEMP_0_PIN         PC0   // Analog Input
 #define TEMP_1_PIN         PC1   // Analog Input
 #define TEMP_BED_PIN       PC2   // Analog Input
+#define TEMP_PROBE_PIN     PC1   // Analog Input
```

Comment:
Take a look at pins_BTT_SKR_common.h for how this is handled so you don't end up defining `PC1` to both `TEMP_1_PIN` and `TEMP_PROBE_PIN`: https://github.com/MarlinFirmware/Marlin/blob/b14c933171b50c87808367bf14872fb7acced92d/Marlin/src/pins/lpc1768/pins_BTT_SKR_common.h#L66-L68

Reply:
Thanks for the suggestion. I'll update the PR.